MAUI/Maps/vector-layers/circle-layer.md

```diff
@@ -489,4 +489,148 @@ this.Content = maps;
 
 {% endtabs %}
 
-![Circle stroke color](images/circle-layer/circle-stroke-color.png)
\ No newline at end of file
+![Circle stroke color](images/circle-layer/circle-stroke-color.png)
+
+## Animation
+
+You can apply animation for the [`MapCircle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapCircle.html) using the `MapCircleLayer.AnimationDuration` and `MapCircleLayer.AnimationEasing` properties and able to customize the animation curve and duration.
```

Comment:
You can apply animation for the [`MapCircle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapCircle.html) using the `MapCircleLayer.AnimationDuration` and `MapCircleLayer.AnimationEasing` properties and be able to customize the animation curve and duration.

Reply:
Updated the content